README.md

```diff
@@ -1,8 +1,31 @@
 # Redux-cycles
+
+<div align="center">
+  <img src="logo.png" alt="Cycle.js + Redux = <3" />
```

Comment:
I kinda like the red heart. 
Both forms are nice, 
I read the first as `redux plus cycle = love` while the second `redux loves cycle` both make sense so I'm fine with either :)